src/components/WalletConnect.vue

```diff
@@ -52,23 +52,36 @@
 		<a-row class="wallet">
 			<a-card size="small">
 				<a-row align="middle">
-					<a-col :span="20">
-						<a-descriptions :column="1" size="small" class="ellipsis">
-							<a-descriptions-item label="Wallet">{{
-								walletAddress
-							}}</a-descriptions-item>
-							<a-descriptions-item label="Selected Wallet">
-								{{ selectedWallet }}
-							</a-descriptions-item>
-						</a-descriptions>
-					</a-col>
-					<a-col style="margin: auto">
+					<a-col>
+						<a-dropdown>
+							<template #overlay>
+								<a-menu @click="handleAccountSwitch">
+									<a-menu-item v-for="addr in walletAddresses" :key="addr">
+										{{ addr }}
+									</a-menu-item>
+								</a-menu>
+							</template>
+							<a-button>
+								<span v-if="!walletAddresses.length">No Address Found</span>
```

Comment:
we are already in v-else from `!walletaddress` template that means this won't be called ever.

Reply:
No, we are not in v-else. First v-if will be checked where I have put check condition on wallet address length.

Reply:
![image](https://user-images.githubusercontent.com/100185149/167368162-8a78d253-c4ac-4608-8f8d-414847ae2def.png)

The topmost template has the v-if for `walletaddress` length, thus the second template won't show if `walletaddress` is empty, also I have tested all wallets and I have seen it doesn't allow you to grant access unless there is atleast one account available in the network, so I think we don't need the check

Reply:
Ok got it.